CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
-## 0.4.1 (Target Date May 1st, 2023)
+
+## 0.5.1 (Target Date May 2nd, 2023)
```

Comment:
Should this be 0.4.2 or is it worthy of a minor bump?

Reply:
tbh I was just preserving the version @smoy was using